### PR TITLE
Added .env

### PR DESCRIPTION
**Pull request by ErFUN-KH**
Now it's easier to use

### CODE REVIEW

**Comment by fzerorubigd** [comment]
I am not sure if I want to add .env (with a requirement for an external tool to load the env) to the project.

**Comment by ErFUN-KH** [comment]
Sorry, I forget to include .env, I think it will work without any external tools now.
if it doesn't work for you, please tell me whats your os, I'll fix it.

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,12 @@
+include .env
+
 export ROOT:=$(realpath $(dir $(firstword $(MAKEFILE_LIST))))
 export BIN:=$(ROOT)/bin
 export GOBIN:=$(BIN)
 export PATH:=$(BIN):$(PATH)
-export PROJECT=engine
 export PROTOTOOL_VERSION=1.8.0
 export DOKKU_HOST=elbix.dev
```
Comment (fzerorubigd) [inline Makefile:8]: Move this one into .env

**Review by fzerorubigd — CHANGES_REQUESTED** [review]
This breaks the circleci test, since those scripts are using the make file and the make. 

Another issue is the `.env`, it is a defacto name for all env loaders (like direnv) and also docker-compose , (which I have plan to migrate the development from vagrant to it)

**Pull request closed without merging**